camel/retrievers/vector_retriever.py

```diff
@@ -92,6 +92,14 @@ def process(
         """
         if isinstance(content, Element):
             elements = [content]
+        elif isinstance(content, IO):
+            from unstructured.partition.auto import partition
+
```

Comment:
I think we need to use unstructured io by `from camel.loaders import UnstructuredIO
`, maybe we also need to do some polish in `UnstructuredIO`